src/components/Book.js

```diff
@@ -1,9 +1,13 @@
 import PropTypes from 'prop-types';
 import { useDispatch } from 'react-redux';
 import { fetchDeleteBooks, fetchGetBooks } from '../redux/books/booksSlice';
+import style from './Book.module.css';
+import loading from '../assets/images/loading.png';
 
 const Book = (props) => {
-  const { id, author, title } = props;
+  const {
+    id, author, title, category,
+  } = props;
   const dispatch = useDispatch();
 
   const handleDeleteBook = (id) => {
```

Comment:
- [ ] Hi, I would advise you not to disable linters, you can solve the bugs manually.

- [ ] ESLint is a tool for identifying and reporting patterns found in ECMAScript/JavaScript code, with the goal of making code more consistent and avoiding bugs.


![Capture](https://user-images.githubusercontent.com/63972670/224368736-f74f8dba-2da0-49ac-b124-2fccb7091f16.PNG)